frontend/test/metabase/scenarios/organization/timelines-collection.cy.spec.js

```diff
@@ -97,8 +98,7 @@ describe("scenarios > organization > timelines > collection", () => {
 
       cy.findByLabelText("Event name").type("RC1");
 
-      // New event modal
-      cy.get(".Modal").within(() => {
+      getModal().within(() => {
```

Comment:
fyi, there is also a [shared `modal` selector](https://github.com/metabase/metabase/blob/53effa4a5e44529e49b6cba936054248d2c5e3fe/frontend/test/__support__/e2e/helpers/e2e-ui-elements-helpers.js#L10) for e2e tests

Reply:
It's a custom modal so this helper doesn't work